pkg/apis/performanceprofile/v2/performanceprofile_validation.go

```diff
@@ -127,7 +127,7 @@ func (r *PerformanceProfile) validateCPUs() field.ErrorList {
 		}
 
 		if r.Spec.CPU.Isolated != nil && r.Spec.CPU.Reserved != nil {
-			cpuLists, err := components.NewCPULists(string(*r.Spec.CPU.Reserved), string(*r.Spec.CPU.Isolated))
+			cpuLists, err := components.NewCPULists(string(*r.Spec.CPU.Reserved), string(*r.Spec.CPU.Isolated), string(*r.Spec.CPU.Offlined))
```

Comment:
What will happen if `r.Spec.CPU.Offlined` is `nil` ? I think you will need a more selective check and list creation here.

Reply:
right, refactoring

Reply:
add a fix. Not sure if it is the most clean one but... Suggestions are welcome